fedimint-core/src/bitcoin_migration.rs

```diff
@@ -8,6 +8,11 @@ pub fn bitcoin29_to_bitcoin30_public_key(pk: bitcoin::PublicKey) -> bitcoin30::P
         .expect("Failed to convert bitcoin v29 public key to bitcoin v30 public key")
 }
 
+pub fn bitcoin30_to_bitcoin29_public_key(pk: bitcoin30::PublicKey) -> bitcoin::PublicKey {
+    bitcoin::PublicKey::from_slice(&pk.to_bytes())
+        .expect("Failed to convert bitcoin v30 public key to bitcoin v29 public key")
+}
+
```

Comment:
nit: ideally this code move wouldn't be here, likely a rebase artifact.

Reply:
Not a rebase artifact - we're moving this out of the `tests` module now that we actually need it in prod code. Previously we were just calling it in a few tests.